### PR TITLE
send button alignment and footer overflow

### PR DESCRIPTION
**Pull request by deepakpathania-zz**
Alignment fixed on both lg and xs screens without the hard coded margins.

### CODE REVIEW

**Comment by deepakpathania-zz** [comment]
Footer overflow also fixed resolving #64

**Comment by handasolo** [comment]
Will take a look tonight. Closing your previous PR.

**Comment by handasolo** [comment]
screen shot 2016-02-22 at 19 24 35

**Comment by handasolo** [comment]
Found the reason why this button is behaving so erratically. Watch the line comments.

File: index.html [hunk 1]
```diff
@@ -384,9 +384,9 @@ <h2 style="color:white" class="section-heading">Contact Us</h2>
                     <span class="error" id="error_message"></span>
                   </div>
                   <div class="form_group">
-                    <div class="row">
+                    <div class="row" style="margin: 0 auto; width: 656px; text-align: center;" >
```
Comment (handasolo) [inline index.html]: This is totally unnecessary styling. Firstly, always try to keep your markup, your styling, and your styling separate. So, no style tags. Secondly, I don't see how this is helping our case.

File: index.html [hunk 2]
```diff
@@ -384,9 +384,9 @@ <h2 style="color:white" class="section-heading">Contact Us</h2>
...
                       <span id="success" class="col-xs-4"></span>
```
Comment (handasolo) [inline index.html]: This is what I missed in the first review, This being a row out of which 4 columns are occupied by this, `center-block` alone doesn't seem to be working. And, without center block we get the deviation in different screens that we were looking at.

Comment (handasolo) [inline index.html]: So essentially, just make this success message appear after the send button in a separate row. That way both of them can use the `center-block` class.

File: index.html [hunk 3]
```diff
@@ -384,9 +384,9 @@ <h2 style="color:white" class="section-heading">Contact Us</h2>
...
-                      <input type="submit" name="send" id="send" value="Send" class="col-sm-8 btn btn-lg send-button center-block" >
+                      <input type="submit" name="send" id="send" value="Send" class="col-sm-8 col-xs-pull-5 col-sm-pull-0 col-sm-push-0 btn btn-lg send-button center-block" >
```
Comment (handasolo) [inline index.html]: Get rid of all the `col` class here after you get the success message in a separate row.

**Comment by deepakpathania-zz** [comment]
I tried to reproduce what you just said but it didn't seem to fix the issue. Maybe, I misunderstood your instructions. Here is what I did. 
https://gist.github.com/deepakpathania/4f3c4efc8ebfd9e7ca96 
Please have a look and tell me what I did wrong.

**Comment by handasolo** [comment]
@deepakpathania Remove `col-sm-8` and you'll be just fine.
P.S. Make sure to try and submit the form and see where the success message is rendered.

**Comment by handasolo** [comment]
Also, when you're done with this make sure to squash your commits.

**Comment by deepakpathania-zz** [comment]
I am done with the final changes. I tried re basing but I was unable to squash commits on a remote branch, tried googling but couldn't squash all the commits into one. Help would be appreciated :)

**Comment by handasolo** [comment]
@deepakpathania Probably because you updated your fork before squashing your commits. Make a fresh fork and deploy the changes.

**Comment by deepakpathania-zz** [comment]
Will do. Closing this one.

**Pull request closed without merging**